client/src/pages/UserProfile.js

```diff
@@ -0,0 +1,234 @@
+  },
+  bioContainer: {
+    display: "flex-column",
+    alignSelf: "center  ",
+  },
+  bioHead: {
+    color: "grey",
+    paddingBottom: "1em",
+  },
+  biography: {
+    color: "grey",
+    marginBottom: "0.75em",
+  },
+  cardFavorites: {
+    // border: "3px solid red",
+    alignSelf: "center",
+    display: "flex",
+    flexDirection: "column",
+    justify: "space-between",
+  },
+  favoritesList: {
+    // border: "3px solid black",
+    display: "flex",
+    flexDirection: "row",
+    ["@media (max-width:600px)"]: {
+      flexDirection: "column",
+      marginBottom: "1em",
+    },
+    padding: 0,
+  },
+  favoriteCuisine: {
+    height: "2em",
+    display: "flex",
+    width: "auto",
+    ["@media (max-width:800px)"]: {width: "auto"},
+    background: "#FF743D",
+    marginRight: "0.75em",
+    marginTop: "0.75em",
+  },
+  cuisineText: {
+    color: "white",
+    alignSelf: "center",
+    fontStyle: "bold",
+  },
+  // Footer
+  mapImage: {
+    width: "100%",
+  },
+  Typography: {
+    fontFamily: "Montserrat, sans-serif",
+  },
+}));
+
+export default function ProfilePage(props) {
+  const classes = useStyles();
+  const bioTitle = "Hi everyone!";
+  const bio = "I'm a foodie and I love to eat healthy tasty meals. Also I'm a mom of two kids";
+  return (
+    <Grid container xs={12} className={classes.outerGrid}>
+      <Grid container xs={12} lg={8} className={classes.innerGrid}>
+        {/* heading container */}
+        <Grid container xs={12} className={classes.cardHeader}>
+          <Grid item lg={4} xs={12} className={classes.headerLeftContainer}>
+            <Grid
+              container
+              display="flex"
+              direction="column"
+              align="center"
+              justify="space-evenly"
+              className={classes.cardHeadLeft}
+            >
+              <Avatar
+                src={profile_pic}
+                className={classes.cardHeadAvatar}
+                borderColor="white"
+              />
+              <Typography variant="h6" className={classes.cardHeadName}>
+                Christine Wilconson
+              </Typography>
+              <Typography className={classes.cardHeadLocation}>
+                Ontario, Canada
+              </Typography>
+              <Button className={classes.cardSendMessage}>Send Message</Button>
+            </Grid>
+          </Grid>
+
+          <Grid item lg={8} xs={12} className={classes.headerRightContainer}>
+            {/* heading */}
+            <Grid container xs={12} className={classes.cardHeadRight}>
+              <Grid className={classes.bioContainer} xs={8}>
+                <Typography
+                  component="h6"
+                  xs={12}
+                  md={4}
+                  className={classes.cardBio}
+                >
+                  <Box paddingBottom="1em" fontWeight="fontWeightBold">
+                    ABOUT ME:
+                  </Box>
```

Comment:
We generally wanna avoid using Box and use Grid instead. Also whenever you have text, put it in Typography and use the `variant` prop (And if necessary, styling and things like `<strong>`) to change it. This is to stay consistent with styling across the app.